File: program2.java
import java.util.*;

class program2 {
    public static void main(String Arg[]) {
        Scanner sobj = new Scanner(System.in);

        int A = 0, B = 0;
        int iAns = 0;

        System.out.println("Enter the number A:");
        A = sobj.nextInt();

        System.out.println("Enter the number B:");
        B = sobj.nextInt();

        iAns = A + B;

        System.out.println("Addition of two numbers:" + iAns);
    }

}

File: program2.cs
using System;

namespace hello
{
    class program
    {
        static void Main(string [] args)
        {
           // int A;
           
            
            Console.WriteLine("Enter the Number A:");
            string inputA =Console.ReadLine();
            int NumberA=int.Parse(inputA);

            Console.WriteLine("Enter the Number B:");
            string inputB =Console.ReadLine();
            int NumberB=int.Parse(inputB);

            int NumberC=NumberA+NumberB;

           Console.WriteLine("Addition of two numbers:"+NumberC);

            Console.ReadLine();

        }
    }
}